src/activity-logger/background/log-page-visit.test.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env jest */
+/* eslint import/namespace: "off" */
+
+import maybeLogPageVisit from 'src/activity-logger/background/log-page-visit'
+import * as storePage from 'src/page-storage/store-page'
+import db from 'src/pouchdb'
+
+afterAll(async () => {
+    await db.destroy()
+})
+
+describe('maybeLogPageVisit', () => {
```

Comment:
Maybe add a test for the call to `storeVisit`.
I would also suggest to extract a `logPageVisit` function from `maybeLogPageVisit`. E.g.:

```js
export async function maybeLogPageVisit({tabId, url}) {
    // Check if we want to log this page (hence the 'maybe' in the name).
    if (isWorthRemembering({url})) {
        await logPageVisit({tabId, url})
    }
}

export async function logPageVisit({tabId, url}) {
    // The time to put in documents.
    const timestamp = Date.now()

    // TODO first try to extend an existing visit instead of logging a new one.

    // First create an identifier for the page being visited.
    const {page, finalPagePromise} = await reidentifyOrStorePage({tabId, url})
    // Create a visit to this page.
    const visit = await storeVisit({page, url, timestamp})

    // TODO possibly deduplicate the visit if it was to the same page after all.
    void (finalPagePromise, visit)
}
```